.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Checklist
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**PR Buddy:** _@ mention a team member_
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide historical context and customer-specific info._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any AMD hardware gotchas._
+
+**Additional Documentation:**  
+_What else should the reviewer know?_
+
+**Test Cases:**
+- [ ] _List here_
+
+## Approval Checklist
+- [ ] PR Buddy: Test functionality
+- [ ] PR Buddy: Test performance
+- [ ] PR Buddy: Verify accurate documentation by testing listed steps
```

Comment:
Some PRs fix customer platform that we may not have access. There needs some flexibility here

Reply:
Eed-to-end ML workload test steps can be very complicated. Again maybe better use link to JIRA, which is supposed to have detailed reproduction steps.

Reply:
Test functionality and performance is too much burden for "PR Buddy". This task should be done by CI steps.

Reply:
The direction here is to be more meticulous. Ensure that we document all relevant test cases, and also document any of those that were skipped, and explain why.

Reply:
If a customer platform is involved, do not mention in the PR but document internally what the system was and how it was tested upon.

Reply:
End-to-end is out of scope for this stage of development.

Reply:
Rather than burden with the reviewer with testing functionality and performance, they must ensure that those steps have been covered.